code_src/staking/dot/bounder/boundModule.py

```diff
@@ -0,0 +1,181 @@
+import sys
+from substrateinterface import Keypair
+from substrateinterface.exceptions import SubstrateRequestException
+from Logger import myLogger
+
+
+class Bounder(object):
+    def __init__(self):
+        self.__name = "Bounder"
+        self.logger = myLogger(self.__name)
+        self.logger.info("Start Bounder Program.")
+        self.call_module = "Staking"
+
+    def getKeyFromSeed(self, seed):
+        try:
+            return Keypair.create_from_mnemonic(seed)
+        except ValueError as e:
+            self.logger.error(e)
+            sys.exit(0)
+
+    def bond(self, active_substrate, controller_addr, value, reward_address=""):
+        self.logger.info("start bond script.")
+        """
+        {'call_name': 'bond',
+        'call_args': [{'name': 'controller', 'type': 147, 'typeName': '<T::Lookup as StaticLookup>::Source', 'docs': []},
+        {'name': 'value', 'type': 51, 'typeName': 'BalanceOf<T>', 'docs': []},
+        {'name': 'payee', 'type': 152, 'typeName': 'RewardDestination<T::AccountId>', 'docs': []}],
+        'documentation': 'Take the origin account as a stash and lock up `value` of its balance. `controller`will
+        be the account that controls it.
+        `value` must be more than the `minimum_balance` specified by`T::Currency`.
+        The dispatch origin for this call must be _Signed_ by the stash account.
+        Emits `Bonded`.
+        # <weight>
+        - Independent of the arguments. Moderate complexity.
+        - O(1).\n- Three extra DB entries.
+        NOTE:Two of the storage writes (`Self::bonded`, `Self::payee`) are _never_ cleaned
+        unless the `origin` falls below _existential deposit_ and gets removed as dust.
+        # </weight>',
+        'module_prefix': 'Staking','module_name': 'Staking', 'spec_version': 9122}
+        :return:
+        """
+        if not reward_address:
+            reward_address = controller_addr
```

Comment:
Can you comment above this function, i.e. line 20, this behavior, i.e. if no reward address is passed in, the default is to set it as the controller address

Reply:
anyway i will comment all the project but i will comment this.
i would like to comment the code at the end.